main.tf

```diff
@@ -326,10 +340,18 @@ module "cloudflare" {
   }
 }
 
+module "hydra" {
+  source      = "github.com/serlo/infrastructure-modules-shared.git//hydra?ref=master"
+  dsn         = "postgres://${module.kpi.kpi_database_username_default}:${var.kpi_kpi_database_password_default}@${module.gcloud_postgres.database_private_ip_address}/hydra"
+  url_login   = "https://de.${local.domain}/auth/hydra/login"
+  url_consent = "https://de.${local.domain}/auth/hydra/consent"
+  namespace   = kubernetes_namespace.community_namespace.metadata.0.name
+  salt        = "1234567890123456789"
```

Comment:
This should probably be a random value or come from the secrets file, right?

Reply:
I don't really remember what this is used for, only that it should never change https://www.ory.sh/docs/hydra/advanced#subject-identifier-algorithms. I guess moving it to the secrets file does make sense..?